Loop/Managers/AppListManager.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  AppListManager.swift
+//  Loop
+//
+//  Created by Dirk Mika on 11.03.24.
+//
+
+import SwiftUI
+import Algorithms
+
+class AppListManager: ObservableObject {
+
+    struct App: Identifiable {
```

Comment:
It might also be interesting to store the app's URL, so that inside the settings window, you would be able to preview the app's icon (rather than just being text)!

Look at `WallpaperView` if you need help loading an image from a file URL :)